packages/core/src/actions/contracts/multicall.ts

```diff
@@ -70,18 +72,27 @@ export async function multicall<Data extends any[] = Result[]>({
         contractInterface,
       })
       const params = Array.isArray(args) ? args : args ? [args] : []
-      const callData = contract.interface.encodeFunctionData(
-        functionName,
-        params,
-      )
-      if (!contract[functionName])
-        console.warn(
-          `"${functionName}" is not in the interface for contract "${addressOrName}"`,
+      try {
```

Comment:
We weren't guarding against potential errors when encoding function data (e.g. a user may provide invalid `params`, causing `encodeFunctionData` to blow up), and this caused `multicall` to always return `[null]`.